idb/postgres/postgres.go

```diff
@@ -397,9 +397,29 @@ func (db *IndexerDb) AddBlock(block *bookkeeping.Block) error {
 // LoadGenesis is part of idb.IndexerDB
 func (db *IndexerDb) LoadGenesis(genesis bookkeeping.Genesis) error {
 	f := func(tx pgx.Tx) error {
+		// check networkdID
+		network, err := db.getMetastate(context.Background(), nil, schema.NetworkMetaStateKey)
```

Comment:
suggest making `getNetworkState()` and `setNetworkState()` similar to `?etImportState()` and `?etMigrationState()`.